modules/Makefile

```diff
@@ -95,6 +95,7 @@ MODULES_TO_DOCUMENT = \
 	standard/Time.chpl \
 	standard/Types.chpl \
 	standard/Version.chpl \
+	standard/Heap.chpl \
```

Comment:
Oh, and this list is in alphabetical order, so please add Heap where it belongs, not just at the end.